cmake/projects/OpenSSL/schemes/url_sha1_openssl_windows_1_1_plus.cmake.in

```diff
@@ -99,7 +112,9 @@ ExternalProject_Add(
     CONFIGURE_COMMAND
     "@HUNTER_MSVC_VCVARSALL@" "@HUNTER_MSVC_ARCH@"
     COMMAND
-    perl Configure "${opt}" ${shared} no-asm "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@" "--openssldir=${openssl_dir}"
+        "@CMAKE_COMMAND@" -P "@HUNTER_PACKAGE_BUILD_DIR@/nasm_path_configure.cmake"
+    COMMAND
+    perl Configure "${opt}" ${shared} "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@" "--openssldir=${openssl_dir}"
```

Comment:
Should be part of `nasm_path_configure.cmake.in` now

Reply:
OK... let's try

Reply:
Just pushed but same error. I used execute_process. Maybe this is the wrong way ?